typescript/agentkit/src/wallet-providers/privyShared.ts

```diff
@@ -32,6 +34,22 @@ type CreatePrivyWalletReturnType = {
   privy: PrivyClient;
 };
 
+/**
+ * Create a Privy client
+ *
+ * @param config - The configuration options for the Privy client
+ * @returns The created Privy client
+ */
+export const createPrivyClient = (config: PrivyWalletConfig) => {
```

Comment:
let's use this function in `createPrivyWallet` before to reduce some duplication